### ISSUE TITLE
[Suggestion] Add brave to browser list

### ISSUE THREAD

**Issue by 3Q6Dev** [jabiel/BrowserPass#13]
Code works great for me, but I have a suggestion, adding brave browser (And other chromium based browsers)
In theory, it should work the same as chrome (except different login data location, etc.) as they are both based on chromium
Ive checked the SQL, it has the same login table etc. so it should work the same.

Heres what would need to be changed:
`private const string LOGIN_DATA_PATH = "\\..\\Local\\BraveSoftware\\Brave-Browser\\User Data\\Default\\Login Data";`

**Comment by simpleperson123** [comment]
Good idea. However, I'm not sure if the decryption would work, because newer versions of Chromium use different decryption method (AES-256-GCM). It should decrypt older logins that are encrypted with DPAPI though.
